contrib/99designs/gqlgen/tracer.go

```diff
@@ -92,7 +92,7 @@ func (t *gqlTracer) InterceptOperation(ctx context.Context, next graphql.Operati
 		response := responseHandler(ctx)
 		if span != nil {
 			var spanErr error
-			if len(response.Errors) > 0 {
+			if response != nil && len(response.Errors) > 0 {
```

Comment:
would it be possible to add a test? just to ensure we don't add a regression in the future 🙏